File: t/AES.java
import javax.crypto.Cipher;
import javax.crypto.spec.SecretKeySpec;
import java.util.Base64;
import java.util.Scanner;

public class AES {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        String key = "1234567890abcdef"; // 16 bytes
        SecretKeySpec secretKey = new SecretKeySpec(key.getBytes(), "AES");

        System.out.print("Enter plaintext: ");
        String plaintext = sc.nextLine();

        // Encrypt
        Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5Padding");
        cipher.init(Cipher.ENCRYPT_MODE, secretKey);
        byte[] encrypted = cipher.doFinal(plaintext.getBytes());

        System.out.println("Encrypted: " + Base64.getEncoder().encodeToString(encrypted));

        // Decrypt
        cipher.init(Cipher.DECRYPT_MODE, secretKey);
        byte[] decrypted = cipher.doFinal(encrypted);

        System.out.println("Decrypted: " + new String(decrypted));
    }
}

File: t/AES.py
# pip install pycryptodome

from Crypto.Cipher import AES
from Crypto.Util.Padding import pad, unpad

key = b'0123456789abcdef'
cipher = AES.new(key, AES.MODE_ECB)

text = b'HelloAESWorld!!'

encrypted = cipher.encrypt(pad(text, 16))
decrypted = unpad(cipher.decrypt(encrypted), 16)

print(encrypted)
print(decrypted)